components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.html

```diff
@@ -169,7 +169,16 @@ <h2 class="metadata-group-heading">Metadata</h2>
           </chef-option>
         </chef-phat-radio>
 
-        <chef-table class="controls-table">
+        <div class="metadata-container err-container">
```

Comment:
I think this should only be shown when there is an error. I am seeing this when a node does not fail.

![image](https://user-images.githubusercontent.com/1679247/88693357-0bddd700-d0b4-11ea-85d2-3c36d3569f0b.png)

Reply:
You are right @lancewf, I had the ngif to make the div only show, but it vanished in my css struggles. I pushed another commit to bring the ngif back. Tested it with a failed node and a padded one and works as expected now. Thank you